course_catalog/constants.py

```diff
@@ -67,8 +67,8 @@ class ListType(Enum):
     to the user as "Your Lists"
     """
 
-    LEARNING_PATH = "Learning Path"
-    LIST = "List"
+    LEARNING_PATH = "learning_path"
+    LIST = "user_list"
```

Comment:
I'm actually changing the types [in another PR](https://github.com/mitodl/open-discussions/pull/2346/files#diff-703dc5834e8b55f9876debd9f94446f0R9), maybe it would be best for me to do it there after this one is merged.

Reply:
ok - so should I leave this as-is?

Reply:
sure, it's fine as is for now

Reply:
ok cool, I'll go ahead and merge :)